generator/testspecs/list_opcodes_directauth.spec.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Contributors to the Parsec project.
```

Comment:
Maybe it's nice to have `.yaml` extensions everywhere?

Reply:
wilco

Reply:
done